layouts/partials/js/expand.js

```diff
@@ -0,0 +1,95 @@
+// Setup all expandable tables on the page.
+var tables = document.querySelectorAll('.p-table--expanding')
+
+/**
+  Toggles the necessary aria- attributes' values on the table panels
+  to show or hide them.
+  @param {HTMLElement} element The tab that acts as the handles.
+  @param {Boolean} show Whether to show or hide the expanded row panel.
+*/
+function toggleExpanded(element, show) {
+  var target = document.getElementById(element.getAttribute('aria-controls'))
+  if (target) {
+    element.setAttribute('aria-expanded', show)
+
+    // Adjust the text of the toggle button
+    if (show) {
+      element.innerHTML = element.getAttribute('data-shown-text')
+    } else {
+      element.innerHTML = element.getAttribute('data-hidden-text')
+    }
+
+    target.setAttribute('aria-hidden', !show)
+  }
+}
```

Comment:
This function (and the others) should be written using the ES6 style declarations. See the `tabs.css` for examples.

```javascript
const toggleExpanded = (element, show ) => { ... }
```